plans/dht/test/create-dht-node.go

```diff
@@ -1,25 +1,19 @@
-package utils
+package test
 
 import (
 	"context"
 
 	"github.com/ipfs/testground/sdk/runtime"
 
-	datastore "github.com/ipfs/go-datastore"
-	libp2p "github.com/libp2p/go-libp2p"
-	host "github.com/libp2p/go-libp2p-core/host"
+	"github.com/ipfs/go-datastore"
+	"github.com/libp2p/go-libp2p"
+	"github.com/libp2p/go-libp2p-core/host"
 	kaddht "github.com/libp2p/go-libp2p-kad-dht"
 	dhtopts "github.com/libp2p/go-libp2p-kad-dht/opts"
 )
 
 // CreateDhtNode creates a libp2p Node and a DHT on top of it
-func CreateDhtNode(ctx context.Context, runenv *runtime.RunEnv) (host.Host, *kaddht.IpfsDHT, error) {
-	// Test Parameters
-	var (
-		bucketSize  = runenv.IntParamD("bucket_size", 20)
-		autoRefresh = runenv.BooleanParamD("auto_refresh", true)
-	)
-
+func CreateDhtNode(ctx context.Context, runenv *runtime.RunEnv, bucketSize int, autoRefresh bool) (host.Host, *kaddht.IpfsDHT, error) {
```

Comment:
I'd encapsulate the parameters in a struct for a cleaner interface. We may want to create a little helper method that binds test params from the RunEnv into a struct. Actually, this already exists: https://github.com/mitchellh/mapstructure. Wanna try integrating it inside the `sdk/runtime` module?

```go
type TestParams struct {
    BucketSize *int 	`mapstructure:"bucket_size"`
    AutoRefresh *bool 	`mapstructure:"auto_refresh"`
}

runenv := runtime.CurrentRunEnv()

var tp TestParams
if err := runenv.DecodeTestParams(&tp); err != nil {
    runenv.Abort(err)
	return
}
```

Reply:
And to deal with defaults, you could have a variable with the defaults, and use mergo to merge the incoming parameters into it.

Reply:
Of course! :D

Reply:
I'm moving this last refactor bit to a separate PR, tracked at https://github.com/ipfs/testground/issues/160, so that all of this code doesn't get blocked until Monday/Tuesday with all the travels and catch up.

Reply:
This way, the next refactor will be already supported by CI #157  :)